### PR TITLE
@xtina_starr => AR-84: Path updates on next button

### PR DESCRIPTION
**Pull request by kanaabe**
There's a little more work we would need to do on the Force side to ensure the direct access + refresh work, but this completes the task of updating the path on the next button.

### CODE REVIEW

File: src/Components/Onboarding/Wizard.tsx [hunk 1]
```diff
@@ -34,19 +29,21 @@ class Wizard extends React.Component<Props, State> {
...
+  onNextButtonPressed = (increaseBy = 1) => {
+    const { currentStep } = this.state
+    const { stepComponents, redirectTo } = this.props
+
+    if (currentStep >= stepComponents.length - 1) {
+      window.location.href = redirectTo
+    } else {
+      const stepIndex = currentStep + increaseBy
+      const nextComponent = stepComponents[stepIndex]
+      window.history.pushState({}, "", `/personalize/${nextComponent.slug}`)
```
Comment (kanaabe) [inline src/Components/Onboarding/Wizard.tsx]: This uses the static variable on each component to push the next path.

File: src/Components/Onboarding/index.tsx [hunk 2]
```diff
@@ -1,10 +0,0 @@
-import React from "react"
-
-import Wizard from "./Wizard"
-
-import Artists from "./Steps/Artists"
-import CollectorIntent from "./Steps/CollectorIntent"
-
-export default () => {
-  return <Wizard stepComponents={[CollectorIntent, Artists]} />
-}
```
Comment (kanaabe) [inline src/Components/Onboarding/index.tsx]: Didn't seem like this file was being used anywhere -- let me know if you'd like me to add it back for some reason.

**Comment by kanaabe** [comment]
@yuki24 It doesn't update the URL in Reaction but I've tested this with Force. Oh is that why you used `window.location` in the post-onboarding redirect?

**Comment by kanaabe** [comment]
@xtina-starr good to merge this guy?

**Comment by xtina-starr** [comment]
Oops sorry @kanaabe. Yes! I actually reviewed this Friday and forgot to give it the 👍

**Comment by kanaabe** [comment]
No worries, thanks! Mergin'

**Pull request merged**